bin/genCameraRepository.py

```diff
@@ -1,351 +0,0 @@
-    @return Camera object
-    """
-    # This is all fragile as the CameraGeomDictionary.paf will go away.
-    policyFile = pexPolicy.DefaultPolicyFile("afw", "CameraGeomDictionary.paf", "policy")
-    defPolicy = pexPolicy.Policy.createPolicy(policyFile, policyFile.getRepositoryPath(), True)
-
-    polFile = pexPolicy.DefaultPolicyFile("obs_cfht", filename)
-    geomPolicy = pexPolicy.Policy.createPolicy(polFile)
-    geomPolicy.mergeDefaults(defPolicy.getDictionary())
-    ampParams = makeAmpParams(geomPolicy)
-    ccdParams = makeCcdParams(geomPolicy, ampParams)
-    ccdInfoDict = parseCcds(geomPolicy, ccdParams, ccdToUse)
-    camConfig = parseCamera(geomPolicy)
-    camConfig.detectorList = dict([(i, ccdInfo) for i, ccdInfo in enumerate(ccdInfoDict['ccdInfo'])])
-    if writeRepo:
-        if outputDir is None:
-            raise ValueError("Need output directory for writting")
-
-        def makeDir(dirPath, doClobber=False):
-            """Make a directory; if it exists then clobber or fail, depending on doClobber
-
-            @param[in] dirPath: path of directory to create
-            @param[in] doClobber: what to do if dirPath already exists:
-                if True and dirPath is a dir, then delete it and recreate it, else raise an exception
-            @throw RuntimeError if dirPath exists and doClobber False
-            """
-            if os.path.exists(dirPath):
-                if doClobber and os.path.isdir(dirPath):
-                    print("Clobbering directory %r" % (dirPath,))
-                    shutil.rmtree(dirPath)
-                else:
-                    raise RuntimeError("Directory %r exists" % (dirPath,))
-            print("Creating directory %r" % (dirPath,))
-            os.makedirs(dirPath)
-
-        # write data products
-        makeDir(dirPath=outputDir, doClobber=doClobber)
-
-        camConfigPath = os.path.join(outputDir, "camera.py")
-        with open(camConfigPath, 'w') as outfile:
-            outfile.write(
-                "#!!!!This file is auto generated.----Do not edit!!!!\n"
-                "#!!!!Edit input file and regenerate with $OBS_CFHT_DIR/bin/genCameraRepository.py\n")
-            camConfig.saveToStream(outfile)
-
-        for detectorName, ampTable in ccdInfoDict['ampInfo'].items():
-            shortDetectorName = shortNameMethod(detectorName)
-            ampInfoPath = os.path.join(outputDir, shortDetectorName + ".fits")
-            ampTable.writeFits(ampInfoPath)
-
-    return makeCameraFromCatalogs(camConfig, ccdInfoDict['ampInfo'])
-
-
-def parseCamera(policy):
-    """
-    Make a CameraConfig from a policy
-    @param policy: Policy object to parse
-    @return CameraConfig parsed from the policy
-    """
-    camPolicy = policy.get('Camera')
-    camConfig = CameraConfig()
-    camConfig.name = camPolicy.get('name')
-    # Using pixel scale 0.185 arcsec/pixel from:
-    # http://arxiv.org/pdf/0908.3808v1.pdf
-    camConfig.plateScale = 13.70  # arcsec/mm
```

Comment:
It would be good to save the reference for the platescale in a readme or something.

Reply:
Added a README.